src/_data/metadata.json

```diff
@@ -15,7 +15,7 @@
   },
   "imprint": {
     "name": "Ola Gasidlo-Braendel",
-    "email": "ola.gasidlo-braendel@accenture.com",
+    "email": "ola@opentechschool.org",
```

Comment:
Should the feed author email (a few lines above) also be changed?

Reply:
Good question, I asked the same and the answer was no because that one should not be so easily retrieved by bots.